### DISCUSSION TITLE
skynet开源交流社区 qq群 188323917 + 微信群

### DISCUSSION THREAD

**Discussion by G0561** [cloudwu/skynet#1157] [Show and tell]
大家好， 今天发现， 以前的生活群被解散了， 好多同学突然要加入skynet 开源交流社区群，这里说明一下，这个群建立于2014年 ，目前人数1284，欢迎广大新手老手 skyneter 加入讨论技术， 对，是讨论技术！群里群规好像比较严格， 浮躁的不喜欢技术的还是不要加的好，明人不说暗话～~lol
qq群号： 188323917 需要的朋友来！  

目前微信群也已经开通，欢迎需要的朋友加入 微信搜索群“skynet 开源交流社区” 或扫描二维码
![sknet](https://user-images.githubusercontent.com/4122118/75212977-a5caec00-57c3-11ea-88b9-9001a1deb350.png)

**Comment by AnyWhereButHere** [comment]
备份用 tg群：https://t.me/skynetgroup

**Comment by xia0lei** [comment]
二维码过期了

**Comment by wsdjeg** [comment]
@G0561 作者wiki里面有自己的群，为什么你要自己重新建一个呢？

https://github.com/cloudwu/skynet/wiki/Community

**Comment by hanxi** [comment]
> @G0561 作者wiki里面有自己的群，为什么你要自己重新建一个呢？
> 
> https://github.com/cloudwu/skynet/wiki/Community

因为那个生活群被封过，多个备份 :smile:

**Reply by ypengju** [reply]
二维码失效了
